README.md

```diff
@@ -320,6 +320,18 @@ For example:
 fetch:
   curl --silent https://app.example.com/services/ping/version
 ```
+<h3 id="kubernetes">Kubernetes</h3>
+
+**<code>kubernetes</code>** allows to specify a Kubernetes namespace and context to deploy to.
+
+For example:
+```yml
+kubernetes:
+  namespace: my-app-production
+  context: tier4
+  template_dir: config/deploy/k8s_templates
```

Comment:
nit: the default if you leave this out is `config/deploy/$ENVIRONMENT` so perhaps we should state that and/or make the example reference an environment.